docs/guides/admin/docs/configuration/https/nginx.md

```diff
@@ -113,7 +113,10 @@ http {
             proxy_redirect          http://$host https://$host;
 
             # Make sure to serve cookies only via secure connections.
-            proxy_cookie_path / "/; HTTPOnly; Secure";
+            proxy_cookie_flags ~ secure httponly;
+            # When using Nginx <1.19.3 replace the above 'proxy_cookie_flags' line
```

Comment:
When using old Nginx versions… update? Do we need fullbacks for older versions? We don't have them at any of the other flags.

Reply:
I totally agree. I added this because of the [concerns](https://github.com/opencast/opencast/issues/2135#issuecomment-743192297) raised by @gregorydlogan

Reply:
Fine with me. Then, let's leave this for now and not close the issue, but add a task to remove the comment with Opencast 12. That gives people almost another year.